onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -354,26 +337,25 @@ bool QkvToContext(
   const T* v = k + total_size;
 
   cublasSetStream(cublas, stream);
-  CublasConfigHelper helper(cublas);
+  CublasMathModeSetter helper(cublas, CUBLAS_TENSOR_OP_MATH);
```

Comment:
CUBLAS_POINTER_MODE_HOST is not handled in CublasMathModeSetter. Does it have impact on performance?